src/index.js

```diff
@@ -838,9 +838,10 @@ const members = {
 };
 
 if (!('attachShadow' in document.createElement('div'))) {
+  const commProto = Comment.prototype;
```

Comment:
Reordered the `prototype` constants to be alpha ordered. Left the node instances alone for now.